hack/terraform/eks/main.tf

```diff
@@ -100,7 +100,7 @@ module "eks" {
   version         = "~>17.24.0"
   cluster_name    = local.cluster_name
   subnets         = module.vpc.public_subnets
-  cluster_version = "1.23"
+  cluster_version = "1.25"
```

Comment:
Could you pin 1.25.5 for EKS too

Reply:
Looks like EKS does not support minor versions. I did a quick try and it is failing. 
We can revisit this later.

```
│ Error: Your query returned no results. Please change your search criteria and try again.
│ 
│   with module.eks.data.aws_ami.eks_worker[0],
│   on .terraform/modules/eks/data.tf line 20, in data "aws_ami" "eks_worker":
│   20: data "aws_ami" "eks_worker" {
```